heaviside/__init__.py

```diff
@@ -42,24 +40,22 @@ def compile(source, region = '', account_id = '', visitors = [], **kwargs):
     """
     try:
         with read(source) as fh:
-            if hasattr(fh, 'name'):
```

Comment:
Not worth changing but it looks like you have the IDE automatically changes single-quoted strings to double-quoted.  Makes it a little hard to know what is a real change.

Reply:
Yeah I started using `ruff` and had format on save turned on. Maybe we should take a pass through the code bases and auto-format for consistency.